docs/tutorials/visibility_simulator.ipynb

```diff
@@ -282,7 +282,7 @@
     "\n",
     "The beam models given to the `beams` parameter can be: i) a `pyuvsim.BeamList` object, ii) a `pyuvdata.UVBeam`, or iii) a list of `pyuvsim.AnalyticBeam`, including its subclasses.\n",
     "\n",
-    "The `hera_sim.beams` module provides analytic models of the HERA antenna through the `PolyBeam`, `PerturbedPolayBeam`, and `ZernikeBeam`  objects (see HERA Memo [#081](http://reionization.org/wp-content/uploads/2013/03/HERA081_HERA_Primary_Beam_Chebyshev_Apr2020.pdf) and [#101](http://reionization.org/manual_uploads/HERA101_Analytic_polarized_beam.pdf), and [Choudhuri et al 2021](https://arxiv.org/abs/2101.02684)). Although these beam models are subclasses of `pyuvsim.AnalyticBeam`, they are currently not regcogniziable by the pyuvsim configuration file parser, and thus can only be used by initializing `ModelData` manually."
+    "The `hera_sim.beams` module provides analytic models of the HERA antenna through the `PolyBeam`, `PerturbedPolayBeam`, and `ZernikeBeam`  objects (see HERA Memo [#081](http://reionization.org/wp-content/uploads/2013/03/HERA081_HERA_Primary_Beam_Chebyshev_Apr2020.pdf) and [#101](http://reionization.org/manual_uploads/HERA101_Analytic_polarized_beam.pdf), and [Choudhuri et al 2021](https://arxiv.org/abs/2101.02684)). These beam models are subclasses of `pyuvdata.analytic_beam.AnalyticBeam`, and so are recognizable by the pyuvsim configuration file parser."
```

Comment:
I suspect that "PerturbedPolayBeam" should be "PerturbedPolyBeam"

Reply:
Yup, good catch!